frontend/src/app/components/sidebar/sidebar.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="card flex justify-content-center">
```

Comment:
bootstrap is currently not in the project so we cant use the bootstrap utility classes.